src/xdp/rx.c

```diff
@@ -1010,6 +1059,8 @@ XdpRxQueueCreate(
     XdpQueueSyncInitialize(&RxQueue->Sync);
     RxQueue->Binding = Binding;
     RxQueue->Key = Key;
+    RxQueue->InterfaceRxCapabilities.ChecksumOffload = TRUE;
+    RxQueue->IsChecksumOffloadEnabled = FALSE;
```

Comment:
We can rely on the zero'd structure implicitly setting this to FALSE, no need to set it explicitly here.

Reply:
Sure, will remove.